scripts.js

```diff
@@ -1,6 +1,8 @@
 // Containers - use those for interaction with them
 const container1 = document.getElementById('container-1');
 const container2 = document.getElementById('container-2');
+let tilesFirstContainer = [];
```

Comment:
do you need those arrays (tilesFirstContainer/tilesSecondContainer), can you use the containers' children instead?